app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,136 @@
+@import 'font-awesome';
+
+* {
```

Comment:
- [ ] I acknowledge that you have worked very hard on this project, but just like the previous reviewer suggested, I recommend that you ensure your design looks like the given design guidelines especially the signup and transactions pages. For the signup page, simply style the form inputs (color, font, layout) as closely as possible to the given design guideline.
## Behance Design
![image](https://user-images.githubusercontent.com/63915024/221377829-38a0a6ba-78a8-42b7-ab01-da59614176c2.png)
## Your Design
![image](https://user-images.githubusercontent.com/63915024/221377839-5488bf26-9d69-4aac-88df-6fc8bbe2fa8d.png)

Reply:
@codepantha,
I implemented it and it's working from my end